src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java

```diff
@@ -214,11 +213,11 @@ public ModelAndView upliftRegister(@ModelAttribute("registerUserCommand") @Valid
                     return new ModelAndView(REDIRECT_RESET_INACTIVE_USER, model);
                 }
 
-                if (ex.getStatusCode().equals(HttpStatus.CONFLICT)) {
-                    msg = "Your account is already activated";
-                } else {
-                    msg += "PIN user not longer valid";
-                }
+                msg += "PIN user not longer valid";
```

Comment:
```suggestion
                msg += "PIN user no longer valid";
```
Or maybe just: msg = "Please try your action again. PIN user no longer valid"

Reply:
Yeah, Fixed it

Reply:
Its supposed to say Please try again incase of like a 500 or something, I think is the login anyway